common/hooks/useScrollVelocity.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useReducer, useEffect } from 'react';
+
+export default offset => {
```

Comment:
I _think_ making this a named `const` and exporting that is what we've done elsewhere – can help with debugging IIRC.